src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -313,6 +313,17 @@ export const NavBar: React.FC<NavBarPropsType> = ({
                 <div>{navbarTitle}</div>
               )}
 
+              <div
+                className="nav-schedule nav-photobooth"
+                onClick={() =>
+                  openUrlInNewTab(
+                    "https://burn.sparklever.se/in/outsnappedphotoboothcamp"
```

Comment:
can extract constant for the URL

Reply:
Why new tab? I don't think that's right.

Reply:
fixed in cab515897d168494475f82a5a0970d9270d88094